src/main/java/seedu/address/ui/MiniCard.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.ui;
+
+import java.util.Comparator;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.address.model.person.Person;
+
+/**
+ * An UI component that displays information of a {@code Person}.
+ */
+public class MiniCard extends UiPart<Region> {
```

Comment:
Sorry for the minor nit but do you think `MiniPersonCard` would be a better name than `MiniCard`?